sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobDownloadHeaders.java

```diff
@@ -24,7 +27,7 @@
      * Instantiates an empty {@code BlobDownloadHeaders}.
      */
     public BlobDownloadHeaders() {
-        objectReplicationSourcePolicies = null;
+        objectReplicationPolicies = null;
```

Comment:
Should I be more explicit with source in the name?

Reply:
Maybe `objectReplicationSourceProperties`? Since there's already another variable to hold the destination policy header if we get back the OR info of a destination blob.

Reply:
changed to ORSourcePOlicies